README.md

```diff
@@ -29,11 +29,18 @@ The only property is an array of options. Each option has these properties:
 {
   id: 'contrast', // Identifier for adjustment type
   name: 'Use increased contrast', // Name of adjustment rendered as label text
-  description: 'Pages will display as high-contrast black and white images.', // More info on adjustment
   active: false, // Checked state of option
 }
 ```
 
+## Events
+
+When an option is changed, the 'visualAdjustmentOptionChanged' event is emitted.
+The detail object on the event object receives the complete array of options to
```

Comment:
How come you went with emitting all the options every time instead of just the one that changed?

Reply:
The `filter` CSS property takes a space separated list of values, and to know all of the values being used you'd need the entire state of the filters. BookReader is also going to expect the entire array of filters.

Reply:
Is the goal to tailor the output for the CSS `filter`? If so, would it make sense to just output that directly?

Reply:
Nvm I see how the configuration works below.